app/services/integration_customers/factory.rb

```diff
@@ -2,8 +2,8 @@
 
 module IntegrationCustomers
   class Factory
-    def self.new_instance(integration:, customer:, subsidiary_id:)
-      service_class(integration).new(integration:, customer:, subsidiary_id:)
+    def self.new_instance(integration:, customer:, subsidiary_id:, **params)
```

Comment:
Probably we could get rid of `subsidiary_id` since it is just used for netsuite. But maybe we can do it in another PR.
I think it should not be part of constructor for xero or Anrok and it should be part of params and used only for netsuite...

Reply:
Yes, but I didn't want to do the refactorgin on thsi branch as that would affect more classes than necessary for this Hubspot feature.